contrib/localdev/README.md

```diff
@@ -0,0 +1,129 @@
+# Running a control plane locally with Podman and KinD
+
+This README provides step-by-step instructions on how to run HyperShift binaries locally using Podman and KinD to test a control plane.
+No images need to be built to run binaries for hypershift operator, control plane operator, and hosted cluster config operator.
+
+## Prerequisites
+
+* Podman
+* Kind
+* CLI tools: jq, oc
+* Valid pull secret with access to CI releases in $HOME/.pull-secret
+
+### Step 1: Install Podman (and make sure you have the latest version)
+
+#### On MacOS you need to initialize and start a podman machine (ideally with 8GB of memory):
+
+```bash
+brew install podman
+podman machine init --memory 8096
+podman machine start
+```
+
+### Step 2: Install KinD
+
+KinD (Kubernetes in Docker) runs local Kubernetes clusters using containers as nodes. You can install KinD by following these steps:
+
+#### On MacOS
+
+```bash
+brew install kind
+```
+
+### Step 3: Create a Kubernetes Cluster with KinD
+
+Now that you have both Podman and KinD installed, you can create a local Kubernetes cluster using KinD. Here's a basic example of creating a cluster:
+
+```bash
+KUBECONFIG=kind.kubeconfig kind create cluster
+```
+
+## Installing and Running HyperShift
+
+### Step 1: Make your local binaries
+
+```bash
+make build
```

Comment:
I don't know if this needs to be explicitly mentioned (thinking of brand new hypershift users trying this), but the user needs to be in the directory where they've downloaded the hypershift repo before running this.